### PR TITLE
add subscription service

### PR DESCRIPTION
**Pull request by rolandpo**
add new subscription service for OUSD analytics reports
visit /reports/subscribe and fill out email -> confirm subscription in confirmation email

### CODE REVIEW

File: eagleproject/core/migrations/0048_day_subscriber_alter_analyticsreport_created_at_and_more.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                ("email", models.EmailField(max_length=254, unique=True)),
+                ("conf_num", models.CharField(max_length=15)),
+                ("confirmed", models.BooleanField(default=False)),
+            ],
+        ),
+        migrations.AlterField(
+            model_name="analyticsreport",
+            name="created_at",
+            field=models.DateTimeField(
+                default=datetime.datetime(2023, 2, 21, 14, 44, 51, 970983)
```
Comment (sparrowDom) [inline eagleproject/core/migrations/0048_day_subscriber_alter_analyticsreport_created_at_and_more.py:35]: `created_at` & `updated_at ` already exist and have the `not null` property. And django ORM should be correctly setting these values without any need of doing it manually. Curious what is the reason for these default values?

File: eagleproject/eagleproject/urls.py [hunk 2]
```diff
@@ -29,11 +29,15 @@
...
+    path("reports/weekly", core_views.report_latest_weekly),
     path("reports/do-monthly", core_views.make_monthly_report),
     path("reports/do-weekly", core_views.make_weekly_report),
     path("reports/do-monthly/<int:month>", core_views.make_specific_month_report),
     path("reports/do-weekly/<int:week>", core_views.make_specific_week_report),
-    #path("reports/test_email", core_views.test_email),
+    path("subscribe", core_views.subscribe, name='subscribe'),
+    path("reports/confirm", core_views.confirm, name='confirm'),
+    path("reports/unsubscribe", core_views.unsubscribe, name='unsubscribe'),
+    path("reports/test_email", core_views.test_email),
```
Comment (sparrowDom) [inline eagleproject/eagleproject/urls.py:40]: probably better comment this one out

**Review by sparrowDom — APPROVED** [review]
Couple of minor comments otherwise LGTM

File: eagleproject/core/migrations/0048_day_subscriber_alter_analyticsreport_created_at_and_more.py (hunk 1 above)
Comment (rolandpo) [reply eagleproject/core/migrations/0048_day_subscriber_alter_analyticsreport_created_at_and_more.py]: removed those bits

**Pull request merged**